src/js/components/DataTable/Footer.js

```diff
@@ -27,15 +27,16 @@ const Footer = forwardRef(
   ) => {
     const pin = pinProp ? ['bottom'] : [];
     const { passThemeFlag } = useThemeValue();
+    const { theme } = useThemeValue();
 
     return (
       <StyledDataTableFooter ref={ref} fillProp={fill} pin={pinProp} {...rest}>
         <TableRow>
           {groups && (
             <TableCell
               plain
-              size="xxsmall"
-              pad="none"
+              size={theme.dataTable.footer.groups.size}
```

Comment:
Im not sure Im following the `groups`

Reply:
Yeah I'm open to changing this to groupBy

Reply:
Pushed up changes

Reply:
looks good thank you